Symfony/CS/Fixer/Symfony/PhpdocScalarFixer.php

```diff
@@ -115,9 +115,11 @@ private function fixScalars(Line $line, $tag)
      */
     private static function normalizeTypes(array $types)
     {
-        foreach ($types as $index => $type) {
-            $types[$index] = self::normalizeType($type);
+        foreach ($types as $index => &$type) {
+            $type = self::normalizeType($type);
```

Comment:
Are you sure ? It is not fixed in the repo. He only fixed the bug with the iteration over tokens (which was not only causing a reference mismatch but a runtime error)